applications/GeoMechanicsApplication/tests/cpp_tests/stub_time_loop_executor.cpp

```diff
@@ -0,0 +1,32 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#include "stub_time_loop_executor.h"
+#include "processes/process.h"
+#include "testing/testing.h"
+
+namespace Kratos {
+
+StubTimeLoopExecutor::StubTimeLoopExecutor(size_t NumberOfExpectedProcesses) :
+        mNumberOfExpectedProcesses{NumberOfExpectedProcesses}
```

Comment:
I'm used to repeating the base class here, so:
```suggestion
StubTimeLoopExecutor::StubTimeLoopExecutor(size_t NumberOfExpectedProcesses) :
        TimeLoopExecutor(),
        mNumberOfExpectedProcesses{NumberOfExpectedProcesses}
```

But perhaps you have a good reason not to? If so, could you please share your thoughts on this?

Reply:
No specific reason. Since only the default constructor is used here from the base-class, do you still think it's necessary?

Reply:
Let's leave it as it is now. We may want to revisit this point when we will write our coding conventions that supplement the Kratos Style Guide.